tests/integrated/test-command-args/runtest

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# Compile command-args
+make || exit 1
+
+# Remove directories which may be left from previous run
+rm -r data
+rm -r test
+
+# Run without any directory, check that an exception is thrown
+echo "=========== Running without input directory =============="
+./command-args 2> stderr.log
+if ! grep -i "exception thrown" stderr.log > output.log ; then
+    echo "FAIL: No exception thrown when missing input directory"
+    exit 1
+fi
+
+# Run tests in "data" directory
+echo
+echo "=========== Running without args in data directory =============="
+
+mkdir data
+cp BOUT.inp data
+./command-args 
+
+# Check that we have a BOUT.settings and BOUT.log.0 file
+
+if ! [ -e "data/BOUT.settings" ] ; then
```

Comment:
`-f` would be a tighter check then `-e` - making sure this is not a directory, symlink, etc ...